composer.json

```diff
@@ -25,45 +25,45 @@
         "ext-dom": "*",
         "ext-json": "*",
         "ext-sqlite3": "*",
-        "acquia/coding-standards": "^0.7.0",
-        "composer/composer": "~2.2.5",
+        "acquia/coding-standards": "^0.8.0",
+        "composer/composer": "~2.3.5",
         "cweagans/composer-patches": "^1.7",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-        "ergebnis/composer-normalize": "^2.0.0",
-        "hassankhan/config": "^2.1",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
+        "ergebnis/composer-normalize": "~2.15.0",
+        "hassankhan/config": "^3.0",
         "mglaman/drupal-check": "^1.3",
-        "myclabs/php-enum": "^1.7",
+        "myclabs/php-enum": "^1.8",
         "oscarotero/env": "^1.2",
-        "php-coveralls/php-coveralls": "^2.2",
-        "php-parallel-lint/php-console-highlighter": "^0.5.0",
-        "php-parallel-lint/php-parallel-lint": "^1.2.0",
-        "phploc/phploc": "7.0.2",
-        "phpmd/phpmd": "^2.6",
-        "symfony/config": "^4.1",
-        "symfony/console": "^4.1",
-        "symfony/dependency-injection": "^4.1",
+        "php-coveralls/php-coveralls": "^2.5",
+        "php-parallel-lint/php-console-highlighter": "^1.0",
+        "php-parallel-lint/php-parallel-lint": "^1.3",
+        "phploc/phploc": "^7.0",
+        "phpmd/phpmd": "^2.12",
+        "stecman/symfony-console-completion": "^0.11.0",
```

Comment:
I distinctly remember us removing this feature from `orca.php` and documentation. It looks like it's getting _added_ here, not just updated. Let's remove it.

And while you're at it, please compare the additions to this file with the removals to make sure we didn't accidentally (re-)introduce any extraneous dependencies.